firefox_puppeteer/firefox_puppeteer/api/appinfo.py

```diff
@@ -2,7 +2,7 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from firefox_puppeteer.base import BaseLib
+from ..base import BaseLib
```

Comment:
I decided to kill all relative imports and use absolute ones. It definitely makes it safer. Thanks for pointing your issues out.